help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -0,0 +1,131 @@
+| Export type      | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the PubMatic Connect destination.                                                                                                                                                                               |
+| Export frequency | **[!UICONTROL Streaming]**      | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+>
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![How to authenticate](/help/destinations/assets/catalog/advertising/pubmatic/authenticate-destination.png)
+
+-   **[!UICONTROL Bearer token]**: Fill in the bearer token to authenticate to the destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination details](/help/destinations/assets/catalog/advertising/pubmatic/destination-details.png)
+
+-   **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+-   **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+-   **[!UICONTROL Data partner ID]**: The data partner ID set up in you PubMatic account for this integration.
+-   **[!UICONTROL Default country code]**: The default country code that should be applied to all identities if none is provided in the profile.
+-   **[!UICONTROL Account ID]**: Your PubMatic Connect account ID.
+-   **[!UICONTROL Account type]**: The account type of your PubMatic platform account. These are the available options:
+    -   PUBLISHER
```

Comment:
Hi @vfalon This is a list of options in a dropdown field, they are sub options for 'Account type'. The capitalisation is that way (in documentation and the actual form) because that is the way they need to be sent to our actual API. I can simply remove the options from the documentation instead if you think that will be cleaner, should still explain it well enough.

Reply:
@daniel-hoffmann thanks for the detailed information on this. If that is the case, we should keep them in the docs. Please add them in this form:

[!UICONTROL PUBLISHER]
[!UICONTROL DEMAND_PARTNER]
[!UICONTROL BUYER]

Reply:
Done - please check again and let me know if any other changes are required.